src/Database/Table/ColumnAccessCache.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Database\Table;
+
+use Nette\Caching\Cache;
+use Nette\Caching\IStorage;
+
+
+class ColumnAccessCache
+{
+	/** @var Selection */
+	protected $selection;
+
+	/** @var Cache */
+	protected $cache;
+
+	/** @var string */
+	protected $generalCacheKey;
+
+	/** @var string */
+	protected $specificCacheKey;
+
+	/** @var mixed of touched columns */
```

Comment:
What means `mixed of`? Is it array or not?

Reply:
There was lot of mess arround mixed properties, so I get rid of them and simplify code in separate commit. Let me know what you think.